python/llm/src/ipex_llm/transformers/models/mistral.py

```diff
@@ -1149,11 +1166,15 @@ def mistral_attention_forward_4_39_original(
         kv_seq_len += 1
```

Comment:
is `kv_seq_len = cache_k.shape[-2]` above correct?

Reply:
If the compress kv is used, `kv_seq_len` will be set to the correct number at Line1172.